### PR TITLE
Notifications padding and styling changes

### PR DESCRIPTION
**Pull request by superfrank**
Padding tweaks and some copy changes. So:

image

becomes:

image

@gtrufitt could you take a look please?

### CODE REVIEW

File: static/src/inline-svgs/notifications-explainer-desktop.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...41-5.345-7.812-5.345-12.898.05-7.555 4.572-13.876 10.791-16.856zm14.851 28.829c0 2.004-1.49 3.546-6.938 3.546-4.779 0-6.423-1.645-6.423-3.443 0-1.336.565-2.98 2.929-2.98h6.834c2.364-.053 3.598 1.13 3.598 2.877zm5.858 1.182a10.173 10.173 0 0 0 1.028-4.419c0-5.036-2.672-7.246-8.376-7.246h-9.147c-.977 0-1.799-.822-1.799-1.696 0-.616.514-1.336 1.182-1.85 1.336.462 2.518.514 4.163.514 6.526 0 10.997-3.186 10.946-8.736-.052-3.135-1.388-5.396-3.597-6.731 6.526 2.826 11.203 9.301 11.203 17.01-.053 5.137-2.211 9.762-5.603 13.154zm-9.61-23.382c0 5.036-1.49 5.602-3.34 5.602-1.799 0-3.34-.411-3.34-5.602s1.49-5.858 3.34-5.858c1.902-.001 3.34.822 3.34 5.858z"/><text transform="translate(108.944 101.51)" fill="#333" font-family="'TS3TextSans'" font-size="8.222">Obama plan for trade deals with Asia and</text><text transform="translate(108.944 110.76)" fill="#333" font-family="'ArialMT'" font-size="8.222">Europe in tatters after Senate vote</text><text transform="translate(108.944 89.175)" fill="#333" font-family="'TS3TextSans'" font-size="10.278">The Guardian</text><text transform="translate(108.944 122.089)" fill="#BCBCBC" font-family="'TS3TextSans'" font-size="7.194">theguardian.com</text></svg>
```
Comment (gtrufitt) [inline static/src/inline-svgs/notifications-explainer-desktop.svg]: This seems quite a bit bigger, is that expected?

**Comment by gtrufitt** [comment]
👍

File: static/src/stylesheets/module/preferences/_notifications.scss [hunk 2]
```diff
@@ -214,11 +220,11 @@ $notifications-button-height: 28px;
 
 .live-notifications-explainer__header-blurb {
     @include fs-textSans(3);
-    margin-top: $gs-baseline /2;
+    padding-bottom: $gs-baseline;
 }
 
 .live-notifications-explainer__headline {
-    @include fs-headline(3);
+    @include fs-header(2);
```
Comment (NathanielBennett) [inline static/src/stylesheets/module/preferences/_notifications.scss]: Ah, that's why it didn't look right. Thanks!

**Comment by NathanielBennett** [comment]
:+1: Ship it, @superfrank

**Pull request merged**